src/components/shared/personellcard.json

```diff
@@ -0,0 +1,28 @@
+{
+  "collectionName": "components_shared_personellcards",
+  "info": {
+    "displayName": "PersonellСard",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "title": {
+      "type": "string",
+      "maxLength": 256
+    },
+    "text": {
+      "type": "string",
+      "required": true
+    },
+    "image": {
+      "type": "relation",
+      "relation": "oneToOne",
+      "target": "api::image.image"
+    },
+    "icon": {
```

Comment:
Suggestion:

Let's also add a theme here to control at least text color.

Reply:
yep, added theme to the card